site/docs/releases.md

```diff
@@ -67,16 +67,145 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
 </dependencies>
 ```
 
+### 1.10.0 release
+
+Apache Iceberg 1.10.0 was released on September ??, 2025.
+
+The 1.10.0 release contains bug fixes and new features. For full release notes visit [Github](https://github.com/apache/iceberg/releases/tag/apache-iceberg-1.10.0)
+
+* Deprecation / End of Support
+  - Flink: Remove Flink 1.18 support ([\#13191](https://github.com/apache/iceberg/pull/13191))
+  - Common: Remove public visibility for deprecated methods in DynConstructors and DynMethods since 1.7.0 ([\#13053](https://github.com/apache/iceberg/pull/13053))
+  - AWS, Core, Flink, Parquet: Remove deprecations targeted for 1.10.0 ([\#12909](https://github.com/apache/iceberg/pull/12909))
+* Behavior change
+  - Hive: Throw NoSuchNamespaceException when listing a non-exist namespace ([\#13130](https://github.com/apache/iceberg/pull/13130)). 
+    Previously, an empty list was returned.
+  - Core: fix spec non-confirming field ids for partition stats files ([\#13329](https://github.com/apache/iceberg/pull/13329))
+* Spec
+  - Table: Clarify write requirement to prevent orphaned DVs ([\#13042](https://github.com/apache/iceberg/pull/13042))
+  - Table: Clarify behavior of special geo objects for lower and upper bounds ([\#12956](https://github.com/apache/iceberg/pull/12956))
+  - Table: Add encryption keys ([\#12162](https://github.com/apache/iceberg/pull/12162))
+  - Table: Avoid struct field conflicts in default values ([\#12841](https://github.com/apache/iceberg/pull/12841))
+  - REST: Add row lineage fields ([\#13010](https://github.com/apache/iceberg/pull/13010))
+  - REST: Add encryption keys ([\#12987](https://github.com/apache/iceberg/pull/12987))
+  - REST: remove update to enable row lineage as it is always on for V3 table Support multiple storage credential prefixes ([\#12986](https://github.com/apache/iceberg/pull/12986))
+  - REST: mark 503 as non retryable ([\#13619](https://github.com/apache/iceberg/pull/13619))
+* API
+  - Add table metadata keys for encryption ([\#12927](https://github.com/apache/iceberg/pull/12927))
+  - Add deleteFile to RowDelta API ([\#12861](https://github.com/apache/iceberg/pull/12861))
+  - Expose cleanExpiredMetadata in ExpireSnapshots ([\#13509](https://github.com/apache/iceberg/pull/13509))
+  - Preserve original type for upper/lower bounds in metrics ([\#13695](https://github.com/apache/iceberg/pull/13695))
+  - Fix timestamp(9) with identity partitioning ([\#13746](https://github.com/apache/iceberg/pull/13746))
+  - Add expression factory methods for timestamp literals ([\#13747](https://github.com/apache/iceberg/pull/13747))
+* Core
+  - Fix JDBC catalog race condition with creating table if not exist ([\#13345](https://github.com/apache/iceberg/pull/13345))
```

Comment:
I thought this change was about a failure when the query engine executed `CREATE TABLE IF NOT EXISTS`, but the actual issue is a JDBC catalog initialization failure. 

How about "Fix a race condition in the JDBC catalog during catalog table initialization" or something?

Reply:
agree. will change it to 
```
Fix a race condition in the JDBC catalog initialization with creating system tables
```